packages/settings-library/src/settings_library/email.py

```diff
@@ -17,7 +17,10 @@ class SMTPSettings(BaseCustomSettings):
     SMTP_HOST: str
     SMTP_PORT: PortInt
 
-    SMTP_TLS_ENABLED: bool = Field(False, description="Enables Secure Mode")
+    SMTP_TLS_ENABLED: bool = Field(False, description="Enables TLS Secure Mode")
```

Comment:
what about using an enum? it would per construction remove the fact that both can be enabled.